src/Compilers/CSharp/Portable/Binder/InMethodBinder.cs

```diff
@@ -36,7 +36,12 @@ public IteratorInfo(TypeSymbol elementType, ImmutableArray<Diagnostic> elementTy
         }
 
         public InMethodBinder(MethodSymbol owner, Binder enclosing)
-            : base(enclosing)
```

Comment:
>base(enclosing) [](start = 14, length = 15)

It feels like it would be better to keep one constructor, but adjust the initializer to clear unwanted flags.
```
base(enclosing, enclosing.Flags & ~BinderFlags.AllClearedAtExecutableCodeBoundary)
```

Reply:
It actually makes sense. I'll test it locally and push the changes once I'm finished with the tests.

Reply:
Done.